upgrade/Stage_0_Prerequisites.md

```diff
@@ -376,14 +376,15 @@ example showing how to find the IUF activity.
    +-----------------------------+--------------------------------------+--------------------------------------+-----------------------------+----------------------------+
    | master-secure-kubernetes    | c1bcaf00-109d-470f-b665-e7b37dedb62f | a22fb912-22be-449b-a51b-081af2d7aff6 | management-22.4.0-csm-x.y.z | Management_Master          |
    | worker-secure-kubernetes    | 8b1343c4-1c39-4389-96cb-ccb2b7fb4305 | 241822c3-c7dd-44f8-98ca-0e7c7c6426d5 | management-22.4.0-csm-x.y.z | Management_Worker          |
-   | storage-secure-storage-ceph | f3dd7492-c4e5-4bb2-9f6f-8cfc9f60526c | 79ab3d85-274d-4d01-9e2b-7c25f7e108ca | management-22.4.0-csm-x.y.z | Management_Storage         |
+   | storage-secure-storage-ceph | f3dd7492-c4e5-4bb2-9f6f-8cfc9f60526c | 79ab3d85-274d-4d01-9e2b-7c25f7e108ca | storage-22.4.0-csm-x.y.z    | Management_Storage         |
    +-----------------------------+--------------------------------------+--------------------------------------+-----------------------------+----------------------------+
    ```
 
    Save the name of the CFS configuration:
 
    ```bash
    export CFS_CONFIG_NAME="management-22.4.0-csm-x.y.z"
```

Comment:
For consistency with `STORAGE_CFS_CONFIG_NAME`, it might be nice to name this `MGMT_CFS_CONFIG_NAME` or `KUBERNETES_CFS_CONFIG_NAME`.

```suggestion
   export KUBERNETES_CFS_CONFIG_NAME="management-22.4.0-csm-x.y.z"
```